examples/flux-chat/js/utils/ChatMessageUtils.js

```diff
@@ -21,6 +21,18 @@ module.exports = {
       text: rawMessage.text,
       isRead: rawMessage.threadID === currentThreadID
     };
+  },
+
+  getCreatedMessageData: function(text, currentThreadID) {
```

Comment:
Yeah, that makes more sense. I just copied the name from the original method in the store.